posts/superseding-bpmodels/index.qmd

```diff
@@ -0,0 +1,51 @@
+
+A second key consideration is the plans that the original package author(s) may have had and their views on any future changes. In our case, this was fairly straightforward because the maintainer of the original package was fully involved in the refactoring. Another package author who had made substantial contributions could be reached and gave their approval. More generally, however, bringing all package authors on board with, for example, changes in scope and name is an important step in taking on maintenance of a package and one that should not be neglected.
+
+We also had to consider whether to archive {bpmodels} or allow it to coexist with {epichains}. We decided to keep {bpmodels} accessible to sustain the reproducibility of existing code using the package. The package was moved back to the [epiforecasts GitHub organisation](https://epiforecasts.github.io/bpmodels/) where it originated from. We, however, added a [lifecycle badge](https://lifecycle.r-lib.org/reference/badge.html) to communicate the package’s retired status and text in the README about our plans for low maintenance.
+
+Another technical consideration was how to handle version control and commit histories. When forking a package, it’s important to decide whether to retain the commit history. Options include squashing the history to start with a clean slate, which risks losing visibility of past contributions, or tagging the HEAD commit of the original repository and building from there. For {bpmodels}, we chose to keep the history intact to retain the contributions of its original authors.
+
+[Semantic versioning](https://semver.org/) was another key decision point. Since `{epichains}` was not going to be available immediately but would be developed in the open (on GitHub), we needed to consider how to communicate that to potential users. We decided to start at version `0.0.0.9999` to signal an experimental and unstable phase[^3] while iterating on features.
+
+Throughout this process, we drew inspiration from various sources. Hadley Wickham’s reasoning for {reshape2} as a reboot of {reshape} and Nicholas Tierney's reason for renaming `{ggmissing}` to `{naniar}` were helpful. Additionally, a talk at UseR! 2024 entitled ["retiring packages with extensive reverse dependencies"](https://userconf2024.sched.com/event/1c90V/r-evolution-the-retirement-of-r-packages-with-many-reverse-dependencies-edzer-pebesma-university-of-muenster-roger-bivand-norwegian-school-of-economics) offered practical advice.
```

Comment:
Can you link to Hadley's and Nick's reasons in this paragraph as well? I skipped the footnotes initially and wondered where I could read about these reasons.